GetSchwifty/Code Helper/Completer.swift

```diff
@@ -0,0 +1,326 @@
+//
+//  Completer.swift
+//  GetSchwifty
+//
+//  Created by Lennart Wisbar on 03.05.17.
+//  Copyright © 2017 Benjamin Herzog. All rights reserved.
+//
+
+import Foundation
+
+struct Completer {
```

Comment:
Yes, that's exactly the problem with this name. Still I thought it fits better than FormattingHelper. I don't know.